### ISSUE TITLE
Demo isn't working.

### ISSUE THREAD

**Issue by yasngleer** [madhurjain/gopee#13]

**Comment by Omie** [comment]
yeah, this maintainer has given up on it I guess 👎

**Comment by madhurjain** [comment]
> yeah, this maintainer has given up on it I guess 👎

Not really. But the moment I put up a demo on any of the services, it gets taken down due to violation
